docs/source/squashfs.rst

```diff
@@ -0,0 +1,87 @@
+SquashFS
+========
+
+``conda-pack`` can package environments into
+`SquashFS <https://en.wikipedia.org/wiki/SquashFS>`_, a compressed, read-only Linux filesystem.
+These filesystems can then be mounted directly, without decompressing them first.
+This allows benefiting from compressed storage, without the decompression step necessary for the
+``zip`` and ``tar.gz`` formats.
+
+Packing
+-------
+Packing environments into SquashFS works on MacOS and Linux.
+You will need `squashfs-tools <https://github.com/plougher/squashfs-tools>`_, more specifically
+the ``mksquashfs`` command.
+``squashfs-tools`` can be installed via ``conda install -c conda-forge squashfs-tools``.
```

Comment:
This is also often already available by the Linux distribution of the user (and probably the required way to install that if you use `conda-pack` from `defaults`). Thus: Can you add which Ubuntu packages are needed? This is probably the most common platform.